src/error.rs

```diff
@@ -49,6 +49,8 @@ pub enum Error {
         /// Supported types are `string`, `int`, `float`, `bool`, `color`, `file` and `object`.
         type_name: String,
     },
+    /// WangId not properly formated. Stores the wrongly parsed String.
+    InvalidWangIdEncoding(String),
```

Comment:
This member should be named instead, a String is not very informative of what it represents in this context.

Reply:
Something like:
```rust
    /// WangId not properly formated.
    InvalidWangIdEncoding {
        /// Stores the wrongly parsed String.
        readed_string: String,
    },
`

Reply:
Sure, but it should be "formatted" and "read" instead of "readed".